app/views/courses/show_professors.html.slim

```diff
@@ -42,7 +42,7 @@
                     span GPA
                   .row.course-rating-row
                     - if stats.gpa
-                      h4.course-rating = stats.gpa.round(2)
+                      h4.course-gpa = stats.gpa.round(2)
```

Comment:
Is it not standard to have classes for elements that appear more than once and ids for elements that only appear once? For the sake of compact styles how about giving it two classes `course-rating` and `gpa`